src/GT.UI/Controllers/ListingsController.cs

```diff
@@ -67,52 +78,36 @@ public async Task<IActionResult> PutListing(string id, Listing listing)
 			}
 
 			return NoContent();
+
 		}
 
 		// POST: api/Listings
 		// To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
 		[HttpPost]
-		public async Task<ActionResult<Listing>> PostListing(Listing listing)
+		public async Task<ActionResult<Listing>> PostListing(ListingDTO listing)
 		{
-			_context.Listings.Add(listing);
-			try
-			{
-				await _context.SaveChangesAsync();
-			}
-			catch (DbUpdateException)
-			{
-				if (ListingExists(listing.Id))
-				{
-					return Conflict();
-				}
-				else
-				{
-					throw;
-				}
-			}
-
+			await _ListingService.AddAsync(listing, _userManager.GetUserId(User));
 			return CreatedAtAction("GetListing", new { id = listing.Id }, listing);
```

Comment:
listing.Id is unassigned? Do we manage Guid.ToString() in BLL?

Reply:
That was my assumption but needs to be discussed once services are up and running